### PR TITLE
add examples and epilogue to cli commands

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

Additional information to the CLI command interface.

* `wdio --help` (before/after)

Screen Shot 2019-10-08 at 11 14 46 AM
Screen Shot 2019-10-08 at 11 35 59 AM

* `wdio config --help` (before/after)

Screen Shot 2019-10-08 at 11 16 35 AM
Screen Shot 2019-10-08 at 11 16 49 AM

* `wdio install --help` (before/after)

Screen Shot 2019-10-08 at 11 17 32 AM
Screen Shot 2019-10-08 at 11 17 47 AM

* `wdio repl --help` (before/after)

Screen Shot 2019-10-08 at 11 18 32 AM
Screen Shot 2019-10-08 at 11 18 45 AM

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

Tests will be added by @baruchvlz

### CODE REVIEW

**Comment by christian-bromann** [comment]
> Are you going to add tests?

Yes, it will be happen as part of #4570

**Comment by mgrybyk** [comment]
got it, let's wait for the tests then otherwise build will start failing

File: packages/wdio-cli/src/index.js [hunk 1]
```diff
@@ -1,39 +1,65 @@
...
     const params = { ...argv.argv }
-    if (!params._.find((param) => SUPPORTED_COMMANDS.includes(param))) {
-        params.configPath = path.join(process.cwd(), params._[0] || DEFAULT_CONFIG_FILENAME)
-        params._.push(fs.existsSync(params.configPath) ? 'run' : 'config')
-        return handler(params)
+    const supportedCommands = fs
+        .readdirSync(path.join(__dirname, 'commands'))
+        .map((file) => file.slice(0, -3))
+
+    if (!params._.find((param) => supportedCommands.includes(param))) {
```
Comment (mgrybyk) [inline packages/wdio-cli/src/index.js]: what is `params._` ?

Comment (christian-bromann) [reply packages/wdio-cli/src/index.js]: This is the command that is being passed in. Not sure if there is a different way to access it, I can check

Comment (christian-bromann) [reply packages/wdio-cli/src/index.js]: This is not an internal property and recommended to use by args: https://github.com/yargs/yargs/blob/master/docs/examples.md#and-non-hyphenated-options-too-just-use-argv_

**Comment by mgrybyk** [comment]
Is it expected to have `Not enough non-option arguments: got 0, need at least 1` in the end instead of epilogue?
image

I don't mind, just want to make sure it works as expected.

**Comment by christian-bromann** [comment]
> I don't mind, just want to make sure it works as expected.

What did you executed?

I am currently running manual tests to check functionality of it.

**Comment by mgrybyk** [comment]
`./node_modules/.bin/wdio run`

**Comment by christian-bromann** [comment]
it should ask you if you want to create a config file if you don't have one

**Comment by mgrybyk** [comment]
for me error message like `Not enough non-option arguments: got 0, need at least 1` makes sense.

I wanted to run conf file but didn't proved it, so pretty much fair.

The question was only about epilogue if it should be in the very end in such case or no, both cases works for me

**Comment by christian-bromann** [comment]
I think if the epilogue is always at the end such error message can get lost so I am fine with the observed behavior

**Comment by mgrybyk** [comment]
Can you increase unit test coverage a bit?

**Pull request merged**